src/EVM/Solidity.hs

```diff
@@ -605,8 +605,8 @@ toCode :: Text -> ByteString
 toCode t = case BS16.decodeBase16 (encodeUtf8 t) of
   Right d -> d
   Left e -> if containsLinkerHole t
-            then error "unlinked libraries detected in bytecode"
-            else error $ T.unpack e
+            then error $ internalError "unlinked libraries detected in bytecode"
+            else error $ internalError $ T.unpack e
```

Comment:
Is this always an internal error?

Reply:
Actually, no, you are right :S Thanks! Sorry.  I am changing it to simply "Error:"